### DISCUSSION TITLE
DataDog Sink

### DISCUSSION THREAD

**Discussion by cdhunt** [TomasBouda/PoShLog#36] [Show and tell]
Hi, I've started a [DataDog Sink](https://www.powershellgallery.com/packages/PoShLog.Sinks.DataDog/0.1.0). 

Would you like to host the repo in the PoShLog org or do you prefer not to imply ownership of modules you didn't author?

**Comment by TomasBouda** [comment]
Hi, good job 👍 I think it's ok if you just provide me with url to your repo and I will add that link to PoShLog wiki. I think I've done it similarly with ElasticSearch sink.

**Reply by cdhunt** [reply]
Sounds good. I've pushed up the code. 

https://github.com/cdhunt/PoShLog.Sinks.Datadog
